packages/service-registry-api/src/testdata/query_verifier_returns_verifier_details.golden

```diff
@@ -1,13 +1,13 @@
 {
   "verifier": {
     "address": "verifier",
-      "bonding_state": {
-        "Bonded": {
-          "amount": "1"
-        }
-      },
-      "authorization_state": "Authorized",
-      "service_name": "verifiers"
+    "bonding_state": {
```

Comment:
why did this change? or a better question is why was it broken to begin with?

Reply:
who knows, i guess something is not stable